packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -406,10 +427,21 @@ class Auth {
                 }
             }
         }
-        return await this.queueRequest(
-            () => helpers.loginGuestUser(this.client, {redirectURI: this.redirectURI}),
-            true
-        )
+        this.logWarningOnClient(this.silenceWarnings, slasSecretWarningMsg)
+        return this.clientSecret
+            ? await this.queueRequest(
+                  () =>
+                      helpers.loginGuestUserPrivate(
+                          this.client,
+                          {},
+                          {clientSecret: this.clientSecret}
```

Comment:
Why does `loginGuestUserPrivate` have a different argument list than `loginGuest` .. it looks like both have a second argument that is an object that is perhaps for "options" .. but we chose to put the client secret in an additional third argument.

(note.. I didn't not look at the isomorphic implementation.. so it might be explained in there)

Reply:
the second arguments is for parameters, which I don't think is a suitable place for client secret. See [here](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/140/files#diff-ead6de48bad82a3ff62bb6a6d9c67e35bdddc53aea5836c16607450ad70ca673R170)

Reply:
I see.. if we have thought about making those signatures more similar we would't have as hard of a type making those types for my recommended changes earlier. 🤔

Reply:
🤔 how do you suggest that we can make it similar? we can still make changes to commerce isomorphic before we release another version